code/iomethods.py

```diff
@@ -518,5 +544,132 @@ def decode_time_from_string(timeString):
     print 'Error decoding time string: string does not match a predefined time format'
     return 0
     # **********************************************************************************************************************
+def write_np_array_to_ncdf(lon, lat, t, fileName, dirName):
```

Comment:
to make the function more generic, consider passing a variable list as parameter. That way, the ch4 variable can be passed in as opposed to hard coded, and other parts of the code (existing parts and new parts) can use this function.